src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -180,7 +194,7 @@ public function delete($paths)
 
         foreach ($paths as $path) {
             try {
-                if (! @unlink($path)) {
+                if (! unlink($path)) {
```

Comment:
Why?

Reply:
Because if using supressing errors block catch is never will be executed.

Reply:
So or use supressing errors or catch ErrorException, not both.

Reply:
This method returns true or false to indicate success. This cannot be changed.

Reply:
What is problem? In laravel all warnings transforms to exception